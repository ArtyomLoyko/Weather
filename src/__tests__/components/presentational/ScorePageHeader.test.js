import React from 'react';
import { shallow } from 'enzyme';
import ScorePageHeader from './../../../components/presentational/ScorePageHeader';

describe('ScorePageHeader', () => {
    const props = {
        score: 25,
    };

    const scorePageHeader = shallow(<ScorePageHeader {...props} />);

    it('renders properly', () => {
        expect(scorePageHeader).toMatchSnapshot();
    });
});
