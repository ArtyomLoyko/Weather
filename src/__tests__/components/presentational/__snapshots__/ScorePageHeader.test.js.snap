// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScorePageHeader renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ScorePageHeader
    score={25}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span>
          Congratulations!!!
</span>,
        <br />,
        "Your score:",
        <span>
          25
</span>,
        "!",
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Congratulations!!!",
        },
        "ref": null,
        "rendered": "Congratulations!!!",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      "Your score:",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 25,
        },
        "ref": null,
        "rendered": 25,
        "type": "span",
      },
      "!",
    ],
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span>
            Congratulations!!!
</span>,
          <br />,
          "Your score:",
          <span>
            25
</span>,
          "!",
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Congratulations!!!",
          },
          "ref": null,
          "rendered": "Congratulations!!!",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "Your score:",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 25,
          },
          "ref": null,
          "rendered": 25,
          "type": "span",
        },
        "!",
      ],
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
